jobs/geo/write_building_footprints.py

```diff
@@ -38,7 +38,7 @@ def write_building_footprints(conn):
 
         gdf = gdf[gdf.geometry.geom_type != "GeometryCollection"]
 
-        file_prefix = f"footprints_with_blocks_for_county_fips_{county}"
+        file_prefix = f"footprints_with_tiger_for_county_fips_{county}"
```

Comment:
What do you think about this name, since the files now include county, tract, block, and place data?

Reply:
That makes sense to me as a name change!